plugins/source/gandi/docs/tables/gandi_certificates.md

```diff
@@ -0,0 +1,25 @@
+# Table: gandi_certificates
+
+
+
+The primary key for this table is **id**.
+
+
+## Columns
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|sharing_id|String|
```

Comment:
Should `sharing_id` be part of the PK here?

Reply:
I don't think certificates have clashing ids to warrant for an additional PK.